specs/_features/maxeb_increase/capella.py

```diff
@@ -1021,12 +1025,11 @@ def get_active_validator_indices(state: BeaconState, epoch: Epoch) -> Sequence[V
     return [ValidatorIndex(i) for i, v in enumerate(state.validators) if is_active_validator(v, epoch)]
 
 
-def get_validator_churn_limit(state: BeaconState) -> uint64:
+def get_validator_churn_limit(state: BeaconState) -> Gwei:
     """
     Return the validator churn limit for the current epoch.
     """
-    active_validator_indices = get_active_validator_indices(state, get_current_epoch(state))
-    return max(config.MIN_PER_EPOCH_CHURN_LIMIT, uint64(len(active_validator_indices)) // config.CHURN_LIMIT_QUOTIENT)
+    return max(config.MIN_PER_EPOCH_CHURN_LIMIT * MIN_ACTIVATION_BALANCE, get_total_active_balance(state) // config.CHURN_LIMIT_QUOTIENT)
```

Comment:
Looking at the engineering on this on Lodestar, the churn not being align with EFFECTIVE_BALANCE_INCREMENT would break multiple optimizations we have. Due to JS 53 bit representation of stack integers we treat account weights denominated in EFFECTIVE_BALANCE_INCREMENT instead of GWei so the aggregated network balances fit. Aligning the churn to that value should produce a marginal accounting difference in the overall churn so I would really push for this extra line
```suggestion
    churn = max(config.MIN_PER_EPOCH_CHURN_LIMIT * MIN_ACTIVATION_BALANCE, get_total_active_balance(state) // config.CHURN_LIMIT_QUOTIENT)
    return churn - churn % EFFECTIVE_BALANCE_INCREMENT
```

Reply:
done!